### ISSUE TITLE
paddlepaddle 训练灰度图出错

### ISSUE THREAD

**Issue by qing130** [PaddlePaddle/Paddle#38634]
Python版本：python 3.7
框架版本：PaddlePaddle 2.2.1
反正就是百度AI studio的版本

问题描述：
自己搭建输入单通道图像的模型，然后输入单通道是图像，报错。

"""
test_model= nn.Sequential(
  nn.Conv2D(in_channels=1,out_channels=16,kernel_size=3),
  nn.MaxPool2D(kernel_size=2),
  nn.Conv2D(in_channels=16, out_channels=32, kernel_size=3),
nn.MaxPool2D(kernel_size=2),
nn.Conv2D(in_channels=32, out_channels=32, kernel_size=3),
nn.MaxPool2D(kernel_size=2),
  nn.Flatten(),
  nn.Linear(21632,1024),
    nn.ReLU(),
nn.Linear(1024, 5)
)
paddle.summary(test_model,input_size=(1,1,224,224))
imgSize=224
batchsize=2
model = test_model
#预处理
transform = T.Compose([
    # T.RandomResizedCrop([imgSize, imgSize]),
    # T.RandomHorizontalFlip(),
    # T.RandomRotation(90),
    # T.ToTensor(),
    T.Normalize(mean=0.5, std=0.5)
])
"""

然后自己解决了，解决方法：
在库文件transforms.py中的：
class Normalize(BaseTransform):
中的以下代码注释：
‘‘‘
if isinstance(mean, numbers.Number):
            mean = [mean, mean, mean]
if isinstance(std, numbers.Number):
            std = [std, std, std]
‘‘‘
经过注释以上代码，我在我电脑上是能跑通了，但是我电脑算力不足，想用AI studio，而AI studio却报错！
所以，建议官方源码改成可以兼容灰度图训练的，不仅仅是Normalize，好像其他预处理也是这个问题，比如：RandomResizedCrop

**Comment by WenmuZhou** [comment]
您好，您的反馈对产品非常有价值，我们内部会安排评估，未来有计划尽快回复。在此之前，您可以先自己实现一下相关的预处理op，这个不难的，在github都能搜到对应的代码
